model/event/task_event_test.go

```diff
@@ -102,3 +105,13 @@ func TestLoggingTaskEvents(t *testing.T) {
 		})
 	})
 }
+
+func TestLogManyTestEvents(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+	require.NoError(db.ClearCollections(AllLogCollection))
+	LogManyTaskAbortRequests([]string{"task_1", "task_2"}, evergreen.GithubPRRequester)
```

Comment:
evergreen.GithubPRRequester is technically not a user. It doesn't really matter in this case though.

Reply:
done